File: Selenium/Activities/Activity16.java
package activity;

import java.util.List;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.Select;

public class Activity16 {

	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/selects");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		WebElement dropDown = driver.findElement(By.xpath("//select[contains(@class, 'h-10 ')]"));
		
		Select select = new Select(dropDown);
		
		select.selectByVisibleText("Two");
		select.selectByIndex(3);
		select.selectByValue("four");
		
		List<WebElement> allOptions = select.getOptions();
		System.out.println("All the options are:");
		for(WebElement option : allOptions) {
			System.out.println(option.getText());
		}
		
		driver.quit();

	}

}

File: Selenium/Activities/activity_16.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.select import Select

with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/selects")

    print("Title of the page: ", driver.title)

    dropdown = driver.find_element(By.XPATH, "//select[contains(@class, 'h-10 ')]")

    select = Select(dropdown)

    select.select_by_visible_text("Two")
    select.select_by_index(3)
    select.select_by_value("four")

    allOptions = select.options
    print("All the options are:")
    for option in allOptions: 
        print(option.text)